account_voucher_tax_sat/model/account_voucher_tax_sat.py

```diff
@@ -113,9 +113,9 @@ def action_close_tax(self):
         move_id = self.create_move_sat()
         self.write({'move_id': move_id.id})
 
-        amount_tax_sat = sum([
+        amount_tax_sat = round(sum([
             move_line_tax_sat.credit
-            for move_line_tax_sat in self.aml_ids])
+            for move_line_tax_sat in self.aml_ids]), 2)
```

Comment:
I suggest you to use the round of currency model something [like this ](https://github.com/odoo/odoo/blob/8.0/addons/account/account_invoice.py#L745)